### DISCUSSION TITLE
Multilanguage Support

### DISCUSSION THREAD

**Discussion by biapar** [directus/directus#7862] [Feature Requests]
How to implement a multilanguage data input support?

**Comment by rijkvanzanten** [comment]
Add a translations field 👍🏻 

![CleanShot 2021-09-07 at 14 06 04@2x](https://user-images.githubusercontent.com/9141017/132390733-5fc11764-9885-4f02-a31d-36b6e15e9f44.png)

https://docs.directus.io/concepts/translations/#translations

**Reply by rijkvanzanten** [reply]
BTW, [Discord](https://directus.chat) is a better platform to get these sorts of questions answered quickly 👍🏻

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
